From 10bb52856faf93a7d17165f93c58914da87a45fe Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Wed, 22 May 2019 14:36:20 -0700
Subject: [PATCH 14/18] Revert "Settings: Hide Night Mode suggestion if
 LiveDisplay feature is present"

This reverts commit 33ff15ff98aa5e4fc0d4edc2a3d3ce475053136c.

Change-Id: I367b700318d051e753ac66e350586f9713f7eee3
---
 .../dashboard/suggestions/SuggestionFeatureProviderImpl.java   | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java b/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
index b32120af33..257e8c3902 100644
--- a/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
+++ b/src/com/android/settings/dashboard/suggestions/SuggestionFeatureProviderImpl.java
@@ -86,8 +86,7 @@ public class SuggestionFeatureProviderImpl implements SuggestionFeatureProvider
         } else if (className.equals(WifiCallingSuggestionActivity.class.getName())) {
             return WifiCallingSuggestionActivity.isSuggestionComplete(context);
         } else if (className.equals(NightDisplaySuggestionActivity.class.getName())) {
-            return context.getPackageManager().hasSystemFeature("org.lineageos.livedisplay")
-                    || NightDisplayPreferenceController.isSuggestionComplete(context);
+            return NightDisplayPreferenceController.isSuggestionComplete(context);
         } else if (className.equals(NewDeviceIntroSuggestionActivity.class.getName())) {
             return NewDeviceIntroSuggestionActivity.isSuggestionComplete(context);
         } else if (className.equals(ZenSuggestionActivity.class.getName())) {
-- 
2.17.1

